pkg/kubelet/api/v1alpha1/runtime/api.proto

```diff
@@ -358,9 +369,21 @@ message LinuxUser {
     repeated int64 additional_gids = 3;
 }
 
-message ContainerConfig {
-    // Name of the container.
+// ContainerMetadata holds all necessary information for building unique
+// container name.
```

Comment:
nit, We could further clarify that it's expected that (name, attempt) is unique within a sandbox for the entire lifetime of the sandbox. I'm okay with this level of documentation though.